api/pkg/cli/cmd/search/search.go

```diff
@@ -31,19 +31,19 @@ import (
 const resTemplate = `{{- $rl := len .Resources }}{{ if eq $rl 0 -}}
 No Resources found
 {{ else -}}
-NAME	KIND	CATALOG	DESCRIPTION	TAGS	CATEGORIES
+NAME	KIND	CATALOG	DESCRIPTION	PLATFORMS	TAGS	CATEGORIES
 {{ range $_, $r := .Resources -}}
-{{ formatName $r.Name $r.LatestVersion.Version }}	{{ $r.Kind }}	{{ formatCatalogName $r.Catalog.Name }}	{{ formatDesc $r.LatestVersion.Description 40 }}	{{ formatTags $r.Tags }}	{{ formatCategories $r.Categories }}
+{{ formatName $r.Name $r.LatestVersion.Version }}	{{ $r.Kind }}	{{ formatCatalogName $r.Catalog.Name }}	{{ formatDesc $r.LatestVersion.Description 40 }}	{{ formatPlatforms $r.LatestVersion.Platforms }}	{{ formatTags $r.Tags }}	{{ formatCategories $r.Categories }}
 {{ end }}
 {{- end -}}
 `
 
 const minResTemplate = `{{- $rl := len .Resources }}{{ if eq $rl 0 -}}
 No Resources found
 {{ else -}}
-NAME	KIND	CATALOG	DESCRIPTION
+NAME	KIND	CATALOG	DESCRIPTION	PLATFORMS
```

Comment:
Could you please remove the `PLATFORMS` as recently we had a discussion to keep  the Tags,Categories and Platforms  with `-o wide` flag only